### ISSUE TITLE
Error during Latest Milestone updating

### ISSUE THREAD

**Issue by zarko-tg** [iotaledger/iri#448]
```
java.lang.NullPointerException: null
  at com.iota.iri.model.Transaction.metadata(Transaction.java:68) ~[iri-1.4.1.4.jar:na]
  at com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider.update(RocksDBPersistenceProvider.java:385) ~[iri-1.4.1.4.jar:na]
  at com.iota.iri.storage.Tangle.update(Tangle.java:92) ~[iri-1.4.1.4.jar:na]
  at com.iota.iri.controllers.TransactionViewModel.update(TransactionViewModel.java:124) ~[iri-1.4.1.4.jar:na]
  at com.iota.iri.controllers.TransactionViewModel.setValidity(TransactionViewModel.java:313) ~[iri-1.4.1.4.jar:na]
  at com.iota.iri.BundleValidator.validate(BundleValidator.java:105) ~[iri-1.4.1.4.jar:na]
  at com.iota.iri.Milestone.validateMilestone(Milestone.java:182) ~[iri-1.4.1.4.jar:na]
  at com.iota.iri.Milestone.lambda$init$0(Milestone.java:98) ~[iri-1.4.1.4.jar:na]
  at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_151]

Node:
{
  "appName": "IRI",
  "appVersion": "1.4.1.4",
  "jreAvailableProcessors": 4,
  "jreFreeMemory": 942163904,
  "jreVersion": "1.8.0_151",
  "jreMaxMemory": 5592580096,
  "jreTotalMemory": 2138046464,
  "latestMilestone": "KNMGPBGDEIERSRQQXFUDIIRPNXIXVLWIMKUDKHDCFGFOTK9ZHRUHEAWUVPUNBRWPQPGUIMBXHKFCZ9999",
  "latestMilestoneIndex": 313627,
  "latestSolidSubtangleMilestone": "JMOBNHRNVYVTSOVJUFDQEURHT99ZVKLVCIGHX9WUWDVOFFNSPOOAPATKZJJVAOCULLGKTNXBSIHGZ9999",
  "latestSolidSubtangleMilestoneIndex": 313614,
  "neighbors": 4,
  "packetsQueueSize": 0,
  "time": 1514452799005,
  "tips": 884,
  "transactionsToRequest": 9,
  "duration": 19
}
```

latestSolidSubtangleMilestoneIndex gets stuck on a certain value despite stop/start or instance reboot.

Experienced on a baremetal 8GB | 4 cores | SSD | 24/7.

**Comment by henriklaxhuber** [comment]
Duplicate of #428.

**Comment by DonCryptone** [comment]
This bug is fixed here
https://github.com/iotaledger/iri/commit/8d1e38b76d7cf20f7ef0272190c0a27be18da19f

**Comment by alon-e** [comment]
closing, duplicate of #424 , as noted above - fixed, merged in 8d1e38b and will be in the next release.
